redisUtil.js

```diff
@@ -17,22 +17,45 @@ const getUniqueId = (prefix) => {
     .then(res => res === 0 ? id : getUniqueId(prefix));
   }
 
+const _getMirror = (id, prefix, dataFields) => {
+  return client.hmgetAsync(`${prefix}:${id}`, dataFields)
+    .then((res) => _.zipObject(dataFields, res));
+}
+
+const _setMirror = (prefix, data) => {
+    let id;
+    return getUniqueId(prefix).then(uniqueId => {
+      id = uniqueId;
+      return client.hmsetAsync(`${prefix}:${id}`, data).then((err, res) => id);
+    });
+}
+
```

Comment:
Here I refactored the logic out into two private functions to avoid repeating the same logic. maybe cleaner this way, maybe more confusing, not sure